boards/arm/rddrone_fmuk66/rddrone_fmuk66.dts

```diff
@@ -342,6 +343,26 @@ zephyr_udc0: &usbotg {
 	status = "okay";
 	pinctrl-0 = <&spi1_default>;
 	pinctrl-names = "default";
+	/*
+	 * cs-gpios needs to be populated as per the schematics
+	 * fxos8700  - CS#0
+	 * fxas21002 - CS#1
+	 */
+	cs-gpios = <&gpiob 10 GPIO_ACTIVE_LOW>,
+		   <&gpiob 9 GPIO_ACTIVE_LOW>;
```

Comment:
who uses CS1?

Reply:
@gmarull .. Kindly check again and let us know if you are satisfied

Reply:
missing fxas21002 entry below?

Reply:
Added it back.. please check